packages/react-component-library/playwright.config.ts

```diff
@@ -0,0 +1,123 @@
+import { PlaywrightTestConfig, devices } from '@playwright/test'
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+const config: PlaywrightTestConfig = {
+  testDir: './playwright/tests',
+
+  globalSetup: require.resolve('./playwright/tests/globalSetup'),
+
+  /* Maximum time one test can run for. */
+  timeout: 30 * 1000,
+
+  expect: {
+    /**
+     * Maximum time expect() should wait for the condition to be met.
+     * For example in `await expect(locator).toHaveText();`
+     */
+    timeout: 5000,
+  },
+
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
+
+  maxFailures: process.env.CI ? 10 : undefined,
+
+  /* Set to 1 locally to enable trace recording. */
+  retries: process.env.CI ? 1 : 0,
```

Comment:
If this is set to 1 it will retry on failure and a trace will be collected (left it off locally as it might be annoying).